api/src/main/resources/logback-test.xml

```diff
@@ -0,0 +1,12 @@
+<configuration>
```

Comment:
Isn't this the file you gitignored in a previous PR?

Reply:
Damn it, yes it is.

This is why I should have ignored it before today 😂

Reply:
Removed in latest commit